cpp/ps0006/ps0006_solv.cpp

```diff
@@ -20,7 +20,7 @@ int main()
     cout<<"------------------------------------- \n";

```

Comment:
- [x] `start=0`, `step=1` as default
- [x] up counter works properly when `step` is positive
- [x] down counter works properly when `step` is negative